src/pretalx/orga/views/submission.py

```diff
@@ -90,6 +91,7 @@ def dispatch(self, request, *args, **kwargs):
 
         try:
             submission.confirm(person=request.user, orga=True)
+            submission.log_action('pretalx.submission.confirm', person=request.user, orga=True)
```

Comment:
There is also `pretalx.submission.confirmation`.. sooo is there any easy way to merge them now?

Reply:
Ehh, no, but since this was not in a release, we'll just silently change it and hope for the best. Done in 1da7893c.